.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "nuget"
+    directory: "/gremlin-dotnet"
+    schedule:
+      interval: "daily"
+    target-branch: "3.5-dev"
```

Comment:
I use `3.5-dev` here as the target branch instead of `3.4-dev` as `3.4-dev` still contains the source generation from templates for the GLVs which won't work with Dependabot. But I also think that it's OK to skip updates for `3.4-dev`, given that most users only use the newest version of Gremlin.NET anyway (per nuget.org download statistics), and because we haven't done much updates until now so it's not like this would make anything worse.